save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/projectcollection/CollectionView.kt

```diff
@@ -101,29 +181,63 @@ class CollectionView : AbstractView<CreationViewProps, State>() {
+
+                        tab(state.selectedMenu.name, ProjectTab.values().map { it.name }, "nav nav-tabs mt-3") {
+                            setState {
+                                selectedMenu = ProjectTab.valueOf(it)
+                            }
+                        }
+
+                        when (state.selectedMenu) {
+                            ProjectTab.PUBLIC -> publicProjectsTable {
+                                getData = { _, _ ->
+                                    val response = post(
+                                        url = "$apiUrl/projects/by-filters",
+                                        headers = jsonHeaders,
+                                        body = Json.encodeToString(ProjectFilters(name = "", public = true)),
+                                        loadingHandler = ::classLoadingHandler,
+                                        responseHandler = ::noopResponseHandler
+                                    )
+                                    if (response.ok) {
+                                        response.unsafeMap {
+                                            it.decodeFromJsonString<Array<ProjectDto>>()
+                                        }
+                                    } else {
+                                        emptyArray()
+                                    }
+                                }
+                            }
+
+                            ProjectTab.PRIVATE -> privateProjectsTable {
+                                getData = { _, _ ->
+                                    val response = post(
+                                        url = "$apiUrl/projects/by-filters",
+                                        headers = jsonHeaders,
+                                        body = Json.encodeToString(ProjectFilters(name = "", public = false)),
+                                        loadingHandler = ::classLoadingHandler,
+                                        responseHandler = ::noopResponseHandler
+                                    )
+                                    if (response.ok) {
+                                        response.unsafeMap {
+                                            it.decodeFromJsonString<Array<ProjectDto>>()
+                                        }
+                                    } else {
+                                        emptyArray()
+                                    }
+                                }
                             }
-                        } else {
-                            emptyArray()
                         }
                     }
                 }
             }
         }
     }
 
-    companion object : RStatics<CreationViewProps, State, CollectionView, Context<RequestStatusContext>>(CollectionView::class) {
+    companion object : RStatics<CreationViewProps, CollectionViewState, CollectionView, Context<RequestStatusContext>>(CollectionView::class) {
```

Comment:
Is it ok that we have `CreationViewProps` set for `CollectionView`? Please check that